src-java/opentsdb-topology/opentsdb-storm-topology/src/main/java/org/openkilda/wfm/topology/opentsdb/OpenTsdbTopology.java

```diff
@@ -110,7 +103,55 @@ private void attachInput(TopologyBuilder topology) {
                 .build();
 
         KafkaSpout<String, InfoData> kafkaSpout = new KafkaSpout<>(config);
-        declareSpout(topology, kafkaSpout, OTSDB_SPOUT_ID);
+        MonitoringKafkaSpoutProxy<String, InfoData> proxy = new MonitoringKafkaSpoutProxy<>(kafkaSpout);
```

Comment:
Does it mean that monitoring is always enabled? Does it affect overall performance or create additional load on system components (kafka, opentsdb)?